src/scss/components/events-highlights.scss

```diff
@@ -0,0 +1,97 @@
+.event {
+  $root: &;
+  margin: 0 auto 80px;
+  max-width: 550px;
+
+  &__wraper {
+    display: flex;
+    flex-direction: column;
+    padding: 0 49px;
+  }
+
+  &.feature {
+    padding-bottom: 100px;
+
+    #{$root}__decoration {
+      background-image: url('../static/assets/evnet-forma.png');
+      background-position: right;
+      background-repeat: no-repeat;
+      background-size: auto 100%;
+      bottom: -47%;
+      height: 20%;
+      position: absolute;
```

Comment:
If you are positioning this element absolutely, you need to set the parent position to be relative

Reply:
when I place relative position on the dad the div event__decoration disappears

Reply:
Then, you need to try different values for the `bottom` and `right` properties. Try setting the initial values to `0` so that you can see the element in the page, and from there set the values to the correct ones.

Reply:
![image](https://user-images.githubusercontent.com/65635471/92541475-b7c62680-f20b-11ea-9ff4-0dac60c76cf8.png)

div disappears and in the maximum view of mobile (959px) 
was placed bottom : 0; and 
right : 0;
and it does not appear also put more values to bottom and right

Reply:
Why are you setting the `height` to `0`? That's why it's not visible.

Reply:
Leave it like that for now. Just to unblock this PR and you can continue with the desktop version